### ISSUE TITLE
loss never decreasing

### ISSUE THREAD

**Issue by huan** [zixia/xi-lstm#8]
> 查看了EMBEDDING部分应该没有问题，但是输入到LSTMP layer和LSTMO layer之后对每句话的输出分布极为相似，导致后面对每个单词的标签预测都是同一个值，因此LOSS不降。标准LSTM的LOSS也不降，且标准LSTM layer1对每句话的输出分布不同，到标准LSTM layer2对每句话的输出分布就很相似了

**Comment by huan** [comment]
```
epoch :  0
batch :  0
current loss :  
 154.5876
[torch.FloatTensor of size 1]

batch :  1
current loss :  
 142.3224
[torch.FloatTensor of size 1]

batch :  2
current loss :  
 126.7297
[torch.FloatTensor of size 1]

batch :  3
current loss :  
 106.5865
[torch.FloatTensor of size 1]

batch :  4
current loss :  
 60.0537
[torch.FloatTensor of size 1]

batch :  5
current loss :  
 66.7900
[torch.FloatTensor of size 1]

batch :  6
current loss :  
 44.6394
[torch.FloatTensor of size 1]

batch :  7
current loss :  
 42.1091
[torch.FloatTensor of size 1]

batch :  8
current loss :  
 41.0118
[torch.FloatTensor of size 1]

batch :  9
current loss :  
 40.1179
[torch.FloatTensor of size 1]
```

What changed:
1. use `optim.SGD()` instead of `optim.RMSprop()`
1. add `optimizer.step()` after `loss.backward()`

**Comment by huan** [comment]
Please close this issue if you think it had resolved.

Thanks! :)
